docs/datasheets/danews.md

```diff
@@ -0,0 +1,188 @@
+
+# DaNews
+
+*Version*: 1.0.0
+
+*Homepage*: https://github.com/centre-for-humanities-computing/danish-foundation-models
+
+*license*: Not publicly available.
+
+---
+
+DaNews consist of articles from Danish news and tabloid media from 1 December 2000 to 
+30 April 2021. It consists of ~25 million articles across 1362 news
+sources. Note that newspapers, such as Politiken with multiple outlets, e.g. physically
+and online, are counted as multiple sources.
+DaNews consists of 9.29 billion tokens of which 8.67 Billion (0.93\%) were left after
+quality filtering and deduplication.
+
+## Datasheet
+
+Following the recommendation and framework of [5] we add the following datasheet. 
+
+### Motivation
+
+**For what purpose was the dataset created? Who created the dataset? Who funded the
+creation of the dataset?**
+
+The dataset was created with the purpise of pre-training Danish language models. It was
+created by a team of researchers at Center for Humanities Computing Aarhus (CHCAA) using
+a codebase jointly developed with partners from industry and industry, including KMD,
+Ekstra Bladet, Bristol University and Deepdivr. For more on collaborators on this
+project see the
+[GitHub repository](https://github.com/centre-for-humanities-computing/danish-foundation-models
+).
+
+**Any other comments?**
+
+No.
+
+## Composition
+
+**What do the instances that comprise the dataset represent (e.g., documents, photos,
+people, countries)?**
+
+Instances of the dataset are Danish articles derived from Danish tabloids or news media. 
+
+**How many instances are there in total (of each type, if appropriate)?**
+
+The dataset consists of 25 874 862 documents and 24 826 047 (0.96%) documents in the 
+filtered dataset.
+
+**Does the dataset contain all possible instances or is it a sample (not necessarily
+random) of instances from a larger set?**
+
```

Comment:
```suggestion
```

Reply:
This makes the text appear on the same line when rendered as a markdown. I don't believe that is desirable.